addons/project_issue/migrations/9.0.1.0/noupdate_changes.xml

```diff
@@ -1,28 +1,38 @@
 <?xml version='1.0' encoding='utf-8'?>
 <openerp>
   <data>
-    <!--Changes in noupdate data-->
     <record id="mt_issue_new" model="mail.message.subtype">
-        <field name="description">Issue opened</field>
-        <field name="name">Issue Opened</field>
+      <field name="description">Issue opened</field>
+      <field name="name">Issue Opened</field>
+    </record>
+    <record id="module_install_notification" model="mail.message">
+      <field name="type"/>
+      <field name="model">mail.channel</field>
+      <field name="res_id" ref="mail.channel_all_employees"/>
+      <field name="message_type">notification</field>
     </record>
     <record id="mt_project_issue_new" model="mail.message.subtype">
-        <field name="name">Issue Opened</field>
+      <field name="name">Issue Opened</field>
     </record>
     <record id="mt_project_issue_blocked" model="mail.message.subtype">
-        <field name="sequence">21</field>
+      <field name="sequence">21</field>
+      <field name="default" eval="False"/>
+    </record>
+    <record id="mt_project_issue_stage" model="mail.message.subtype">
+      <field name="default" eval="False"/>
     </record>
     <record id="issue_user_rule" model="ir.rule">
-        <field name="domain_force">['|',
+      <field name="domain_force">['|',
                                             ('project_id.privacy_visibility', 'in', ['portal', 'employees']),
                                             '&amp;',
                                                 ('project_id.privacy_visibility', '=', 'followers'),
                                                 ('message_partner_ids', 'in', [user.partner_id.id]),
                                         ]</field>
-        <field name="name">Project/Issue: employees: portal, employee or (followers and following)</field>
+      <field name="name">Project/Issue: employees: portal, employee or (followers and following)</field>
     </record>
     <record id="mt_issue_ready" model="mail.message.subtype">
-        <field name="name">Issue Ready</field>
+      <field name="description">Issue ready for next sStage</field>
```

Comment:
Are you sure this field is correct (`sStage`)?

Reply:
That's because it is exactly what the message of the record mt_issue_ready has. But I can fix that if you like.

Reply:
Curious that Odoo contains that typo. I think we can fix it, yes.